modules/web/src/core/layout/components/LeftPanel.jsx

```diff
@@ -180,7 +180,9 @@ class LeftPanel extends React.Component {
         const tabs = [];
         const panes = [];
         const { views, onActiveViewChange, ...restProps } = this.props;
-        const activeViewPrev = this.props.activeView || this.context.history.get(HISTORY.ACTIVE_LEFT_PANEL_VIEW);
+        const activeViewPrev = this.props.activeView
+                    || this.context.history.get(HISTORY.ACTIVE_LEFT_PANEL_VIEW)
+                    || views[0].id;
```

Comment:
Will there always be atleast one view ?

Reply:
yeah in our case, explorer, run will be there. But better to fix for all. I have added the fix. Thanks,